charms/mlflow-server/tox.ini

```
@@ -54,4 +54,11 @@ commands =
 
 [testenv:unit]
 commands = 
-	pytest tests
+	pytest tests --ignore={[vars]tst_path}integration
```

it is minor, but rather than say "unit tests are everything except `/tests/integration`" I think we should say "unit tests are `/tests/unit`".  It feels closer to the message our directory structure suggests.  
```suggestion
	pytest {[vars]tst_path}unit
```
Note that I haven't tested this command locally, just run off memory.  It'll show in the CI pretty quick though...

---

will update this. I personally prefers stating the positives too, but [operator-template](https://github.com/canonical/template-operator/blob/main/tox.ini) suggests otherwise. Do you know how others think about this? And would there be push back to change it in operator template?

---

I thought you might have gotten it from there.  Yeah I suggest you open a PR on the template about this.  Actually I could have sworn I already had, but I don't see one.  The positives feel way more fit for purpose than the negatives - I can't think of when the negatives would be better